javascript/javascript1/week4/classExercises/fizzBuzz.js

```diff
@@ -0,0 +1,22 @@
+function fizzBuzz(num1, num2) {
+  const arrayToPrint = [];
+  let currentNum = 0;
+  for (let i = 0; i < 100; i++) {
+    currentNum = i + 1;
+    if (currentNum % num1 === 0 && currentNum % num2 === 0) {
```

Comment:
You can skip `currentNum` all together, and instead set `i` to start from 1. To make sure the loop will execute to 100, you can change the operator from `<` to `<=`.
So at the end lines 4 to 6 could look like this:
```suggestion
  for (let i = 1; i <= 100; i++) {
    if (i % num1 === 0 && i % num2 === 0) {
```